### PR TITLE
Change format flag to default empty

### PR DESCRIPTION
**Pull request by wbxyz**
Also, update the flag docstrings to reflect that the values are
command-specific.

### CODE REVIEW

File: tools/ctl/ctl.go [hunk 1]
```diff
@@ -157,6 +157,8 @@ var getResults = &cobra.Command{
 			return
 		}
 		switch *format {
+		case "":
```
Comment (msuozzo) [inline tools/ctl/ctl.go:160]: you can use a comma instead: `case "", "summary"`

File: tools/ctl/ctl.go [hunk 2]
```diff
@@ -297,14 +299,8 @@ var runBenchmark = &cobra.Command{
...
 		// TODO: Maybe add more format options, or include more data in the csv?
-		case "csv":
-			w := csv.NewWriter(cmd.OutOrStdout())
-			defer w.Flush()
-			for _, v := range verdicts {
-				if err := w.Write([]string{fmt.Sprintf("%v", v.Target), v.Message}); err != nil {
-					log.Fatal(errors.Wrap(err, "writing CSV"))
-				}
-			}
+		case "":
```
Comment (msuozzo) [inline tools/ctl/ctl.go:302]: and here

**Review by msuozzo — APPROVED** [review]

File: tools/ctl/ctl.go (hunk 1 above)
Comment (wbxyz) [reply tools/ctl/ctl.go]: Done

File: tools/ctl/ctl.go (hunk 2 above)
Comment (wbxyz) [reply tools/ctl/ctl.go]: Done

**Pull request merged**